Homepage-options/Landing - redesign-Rohit/index.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Open Source Web Development Project</title>
+    <link
+        rel="stylesheet"
+        href="https://use.fontawesome.com/releases/v5.6.3/css/all.css"
+        integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+        crossorigin="anonymous"
+    />
+  <link
+        href="https://fonts.googleapis.com/css?family=Lato:100,300,400,700,900&display=swap"
+        rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <header class = "header">
+        <div class="header_logo_box">
+            <a href="#">
+                <img src="img/logo.png" alt="logo" class="header_logo">
```

Comment:
Create a folder of `resources` and move `style` folder, and `images` folder in that